parsers/CA_SK.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import Optional
+
+# The arrow library is used to handle datetimes consistently with other parsers
+import arrow
+from requests import Session
+
+TIMEZONE = "America/Regina"
+# Source: https://www.saskpower.com/Our-Power-Future/Our-Electricity/Electrical-System/System-Map
+URL = "https://www.saskpower.com/ignitionapi/Content/GetNetLoad"
+
+
+def fetch_consumption(
+    zone_key: str = "CA-SK",
+    session: Optional[Session] = None,
```

Comment:
```suggestion
    session: Session = Session(),
```
This way we can eliminate the check further down.